src/test/java/core/basesyntax/operations/impl/BalanceHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.operations.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.operations.OperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static final String FRUIT_APPLE = "apple";
+    private static FruitDao fruitsDao;
```

Comment:
```suggestion
    private static FruitDao fruitDao;
```
change it to single maybe?